troubadix/plugins/multiple_re_parameters.py

```diff
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: 2024 Greenbone AG
+
+import re
+from pathlib import Path
+from typing import Iterator
+
+from troubadix.plugin import FileContentPlugin, LinterError, LinterResult
+
+pattern = re.compile(r"script_mandatory_keys.*re\s*:")
```

Comment:
Could be done, but as far as I can tell at least one more loop is needed. With the special script tag I could get the values of the mandatory keys tags and then use a loop to check if more than one has a re parameter.

Reply:
Not sure if comment checking adds any value. There are currently 0 comments on script_mandatory_keys.
```
nhargarter@gb-ho-10 ~/g/troubadix (multiple_re_parameters)> grep "\s*#\s*script_mandatory_keys" -r ~/gb/vulnerability-tests/nasl
nhargarter@gb-ho-10 ~/g/troubadix (multiple_re_parameters) [1]>  
```
checked it against a  manually added comment so regex should be fine
```
nhargarter@gb-ho-10 ~/g/troubadix (multiple_re_parameters)> grep "\s*#\s*script_mandatory_keys" -r nasl
nasl/gb_rhsa_2014_0223.nasl:#  script_mandatory_keys("ssh/login/rhel", "ssh/login/rpms", re:"ssh/login/release=RHENT_5");
```

Reply:
I think it makes sense to add this kind of check to all plugins or directly adjust https://github.com/greenbone/troubadix/blob/8c4aa32ad18ce1527542426a941e1800a519d0c8/troubadix/helper/patterns.py, because I can't think of a reason why we want to ever handle commented-out NASL tags.
But that's probably beyond the scope of this PR.

Reply:
Created Jira issue to cover comment proofing of Troubadix plugins. VTOPS-226
Looked a bit into possible ways to edit the current helper pattern, and one seems maybe usable, but wouldn't trust it not to make many unforeseen errors
Adding a comment sanitized version of the file to the file_context seems to be my favorite option so far.

Reply:
Sounds good to me!